README.md

```diff
@@ -91,6 +147,36 @@ If you prefer a non-Christmas option, you may choose a different song with simil
 
 ---
 
+
+Solution:
+
+??? - though, I'm not sure if this is right? It says in the homework not to just 
+console.log each line.... and I don't think I did that per se, but I defininitly
+ am console.log'ing the solution, ultimately.)
+
```

Comment:
3) As far as console.log() goes, what you've got is fine--the thing to aboid is simply using only console.log() and no logic to write the entire song to the console. I'd strongly suggest getting into the habit of using more human-readable variable names. Single-character names can work well enough in code this short, but even with something like this it's much more readable to use actual words--I don't even like single-character names in my loops, personally. Once you start getting into 100+ line code, you'll definitely want to be in the habit of using longer variable names.